libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -236,6 +237,15 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  private listenForIonModalWillPresent() {
+    if (this.ionModalElement) {
+      this.ionModalElement.addEventListener('ionModalWillPresent', () => {
+        if (this.config.interactWithBackground)
+          this.renderer.addClass(document.body, 'allow-scroll');
+      });
```

Comment:
Consider only adding event listener if needed:
```suggestion
       if (this.config.interactWithBackground) {
         this.ionModalElement.addEventListener('ionModalWillPresent', () => {
          this.renderer.addClass(document.body, 'allow-scroll');
        });
      }
```

Reply:
We need to set the 'allow-modal-scroll' class on will present, otherwise the user will not be able to scroll until the modal is opened.